docs/Reference/Engine-API-Methods.md

```diff
@@ -0,0 +1,230 @@
+---
+description: Engine API methods reference
+---
+
+# Engine API methods
+
+The Engine API allows communication between [consensus and execution layers](../Concepts/Merge.md#execution-and-consensus-clients) of the [post-Merge](../Concepts/Merge.md) Ethereum Client.
+
+See [Ethereum Engine API specification](https://github.com/ethereum/execution-apis/blob/main/src/engine/specification.md) for more information.
+
+## Methods
+
+### `engine_exchangeTransitionConfigurationV1`
+
+Sends to the consensus client to verify the configuration between both clients.
```

Comment:
Sends the execution client configuration?

Reply:
Fixed